c++/src/kj/async-io-win32.c++

```diff
@@ -1114,6 +1114,16 @@ private:
 Promise<void> AsyncInputStream::read(void* buffer, size_t bytes) {
   return read(buffer, bytes, bytes).then([](size_t) {});
 }
+Promise<size_t> AsyncInputStream::read(void* buffer, size_t minBytes, size_t maxBytes) {
```

Comment:
I'm confused -- how did this manage not to be here before and how did the code compile and run for me without it? All the code around this function was clearly copy/pasted from the corresponding surrounding code in async-io.c++. Did I copy the code and then remove this function definition? So strange.

(And yeah, there's a fair amount of code duplication between async-io.c++ and async-io-win32.c++... I will maybe clean it up eventually.)